scripts/extract-rdb.sh

```diff
@@ -2,9 +2,10 @@
 #set -x
 
 function usage_exit() {
-    echo "Usage: $0 <data_path> <output-path>";
+    echo "Usage: $0 <data_path> <output_path> <rdb_cli_path>";
```

Comment:
Can you just make the third parameter optional? That way previous functionality can still work. See line 16/17 below and also that this script is invoked by `extract-rdb-and-validate.sh`